frontend/components/ApplyPage/index.jsx

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { UserProvider } from '../../context/UserContext';
+import { TopBar } from '../TopBar';
+import SectionHeader from '../shared/SectionHeader';
+import Footer from '../footer';
+import Apply from '../apply/index';
+export const ApplyPage = ({ data }) => {
+  const [mentor, setMentor] = useState(false);
+
+  if (data) {
+    const {
+      topBarData,
+      mentorSectionHeaderData,
+      menteeSectionHeaderData,
+      applyData,
+      applyInputFields,
+      footerData
+    } = data;
+
+    return (
+      <>
+        <div className="container">
```

Comment:
don't add the main container to the content__ every section has container div because it affects to the footer